### ISSUE TITLE
Feature request: Don't download again for X days

### ISSUE THREAD

**Issue by hollowaysmith** [Niels-IO/next-image-export-optimizer#185]
The free tier headless CMS I use has 500k image requests every month and with `next-image-export-optimizer` I'm downloading 5k images every time it builds. That means I can build 100 times every month as it downloads the same images every time.

I understand the justification that the source images may have changed since the last build, but I'd find it really useful if it could not re-download the images for a certain amount of days.

So in summary the idea would be to download new image URLs and preexisting image URLs that were at least X days old, but not more recently downloaded images.

**Comment by Niels-IO** [comment]
Hi @hollowaysmith,

You can scrap the first idea. I came up with a simpler mechanism and created #190 for this feature.

**Comment by holloway** [comment]
@Niels-IO Amazing, thanks! Happy to help test it etc if you want
